components/cam/src/dynamics/se/dyn_comp.F90

```diff
@@ -121,9 +121,6 @@ subroutine dyn_init1(fh, NLFileName, dyn_in, dyn_out)
             frontga_idx)
     end if
 
-    ! Initialize dynamics grid
```

Comment:
Why was dyn_grid_init moved down?

Reply:
dyn_grid_init() used to be empty, so it shouldnt change anything to move it, but for fv_nphys>0 there is stuff that needs to be initialized, and leaving it here caused problems, so I had to move it down after some other stuff was initialized. I think  it was the "prim_init" stuff that was the most important.

Reply:
Looking at the change you made to dyn_init do you think it would be better to create a fv_physgrid_init subroutine that can be called at the location that you put dyn_grid_init and leave dyn_grid_init as is?  It's more of a coding aesthetic choice but I think the idea is that all dycores call a dyn_grid_init step right away, and for the SE dycore it's empty because the prim_init stuff kicks in.  But future dycores may take this code as a template and get confused about when they are suppose to do "dyn_init" and if the fv_physgrid stuff has to be a part of dyn_init or is it's own thing.

Reply:
follow-on, the dyn_grid_final could be changed to fv_physgrid_final.

Reply:
I can do that.